react/features/filmstrip/functions.web.js

```diff
@@ -195,7 +195,7 @@ export function calculateThumbnailSizeForResizableVerticalView(clientWidth: numb
 
     return {
         local: {
-            height: DEFAULT_FILMSTRIP_WIDTH,
+            height: Math.floor(availableWidth / interfaceConfig.LOCAL_THUMBNAIL_RATIO),
```

Comment:
Add a local default in case it's missing from interface config?

Reply:
Done